apps/client/hooks/useIntersectionObserver.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable consistent-return */
+import type { RefObject } from "react";
+import { useEffect, useState } from "react";
+
+interface Args extends IntersectionObserverInit {
+  freezeOnceVisible?: boolean;
```

Comment:
```suggestion
  isFreezeOnceVisible?: boolean;
```
is prefix가 붙어도 될 것 같은데요 찬혁님 의견은 어떤가요?

Reply:
야호! 했습니다!

Reply:
[fix(client): boolean 변수 앞에 is 붙이기](https://github.com/toppingskorea/toppings-client/pull/57/commits/525f8c065043b4f7e1c8741a960f584a01c055b3)